source/store/operations_and_invariants/bool_invariants.py

```diff
@@ -342,6 +316,32 @@ def print(graph, precision):
         return Utils.print_boolean(SomeEigenIntegerDQ.calculate(graph), precision)
 
 
+class SomeEigenIntegerS(InvariantBool):
+    name = "Some S-eigenvalue integer"
```

Comment:
The name of some-eigenValue is not changed to some-eigen here either, is this correct?

Reply:
Its Correct. Because the lenght is ok.